lms/static/js/student_account/views/LoginView.js

```diff
@@ -189,11 +189,18 @@
             },
 
             saveError: function(error) {
-                var msg = error.responseText;
+                var errorCode;
+                var msg;
+                // TODO: Add back the following failsafe line once tests are passing,
+                //   or a version of it that adds a default message.
+                // var msg = error.responseText;
```

Comment:
Leftover commented code?

Reply:
Oops.  Updated.  I had a `Reviewer Note` (see above) that had explained, but I didn't realize it was Pending.